### ISSUE TITLE
BUG: 4.0.7 won't run on ubuntu 24.04

### ISSUE THREAD

**Issue by papampi** [Mahdi-zarei/nekoray#30]
Hi,
Can't start 4.0.7 on ubuntu 24.04
When run launcher from extracted zip folder it gives no error and wont start.
When tried to run nekray it gives this error:

```
~/Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/nekoray 
Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/nekoray: /lib/x86_64-linux-gnu/libQt6Core.so.6: version `Qt_6.5' not found (required by Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/nekoray
```

Edit 1:
Tried on Ubuntu 22.04 as well, same error.

Edit 2:
```
sudo apt install qt6-base-dev
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
qt6-base-dev is already the newest version (6.4.2+dfsg-21.1build5).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

Edit 3:
INstalled deb package and that gives same error as well

**Comment by Mahdi-zarei** [comment]
I reverted the upgrade and re-released the builds. This should solve the issue for now. I will try to handle the issues regarding upgrading Qt for linux versions. Thanks for you quick feedback!

**Comment by papampi** [comment]
Nope, still same error:

```
Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/nekoray 
Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/nekoray: /lib/x86_64-linux-gnu/libQt6Core.so.6: version `Qt_6.5' not found (required by Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/nekoray)
```

**Comment by Mahdi-zarei** [comment]
oh I guess i now know where this is coming from...

**Comment by papampi** [comment]
When run with launcher:

```
Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/launcher 
2024/06/16 01:23:02 exe: launcher exe dir: /home/payam/Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray
2024/06/16 01:23:02 Running as launcher
2024/06/16 01:23:02 LD_LIBRARY_PATH=/home/payam/Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/usr/lib QT_PLUGIN_PATH=/home/payam/Downloads/nekoray-4.0.7-2024-06-15-linux64/nekoray/usr/plugins ./nekoray
```

**Comment by Mahdi-zarei** [comment]
The problem is related to the required Qt libraries, apparently the current build system has a manual and inconsistent method for providing them, and i blindly updated the version which caused this issue. I'm rolling back the changes that caused this until I find a solution to properly include the linked libraries. The rollback should not affect any added feature though.

**Comment by Mahdi-zarei** [comment]
I don't currently have access to a Linux dekstop, could you test the 4.0.7 release again?

**Comment by papampi** [comment]
> I don't currently have access to a Linux dekstop, could you test the 4.0.7 release again?

Sure will do it now

**Comment by papampi** [comment]
Noope ... Not fixed. 
Same error.

**Comment by papampi** [comment]
Fixed on latest 4.0.7
